linux-package.sh

```diff
@@ -16,15 +16,14 @@ javapackager -deploy \
     -BappVersion=$ver \
     -Bcategory=Office,Finance \
     -BlicenseType=Apache \
+    -BlicenseFile=LICENSE \
     -Bemail=contact@vinumeris.com \
     -Bicon=client/icons/icon.png \
     -native deb \
     -name lighthouse \
-    -title Lighthouse \
+    -title "Lighthouse: a peer to peer crowdfunding app that uses Bitcoin" \
     -vendor Vinumeris \
     -outdir deploy \
     -appclass lighthouse.Main \
-    -srcfiles $build \
+    -srcdir updates/builds/processed \
```

Comment:
You're welcome. I've tested the .deb package installation on Ubuntu 14.04 LTS using DPKG, but I didn't do any testing with UpdateFX.